### DISCUSSION TITLE
Unique visitor counter could be misleading

### DISCUSSION THREAD

**Discussion by marcoancona** [plausible/analytics#970] [Anything Else]
My understanding is that Plausible does not track users across days. Thus the plot showing visitors across time reports the actual unique visitors for each day but the "global" unique visitor counter on the top of this plot is not actually "unique visitors", is it? 

I believe that being able to know MAU vs WAU vs MAU is a critical piece of information but I can understand the design choice you made. However, I think that counter should at least be renamed or proposed with a disclaimer.

**Comment by metmarkosaric** [comment]
Thanks @marcoancona! The main difference in unique visitors from Google Analytics to Plausible Analytics is that Plausible doesn't use cookies, user ID nor any other persistent identifiers. It is not possible to track people the way Google Analytics does if you don't get the explicit consent from the visitor to do so. 

We have a different method and we reset it once a day. If the same visitor visits your site 5 times in one day we will show that as 1 unique visitor. But if the same visitor visits your site on 5 different days in a month we would show that as 5 unique visitors. This is the whole premise of Plausible and we mention it many places including our data policy: https://plausible.io/data-policy.

**Reply by marcoancona** [reply]
Yes, I understand the mechanism and the reasoning behind it. I am only suggesting being more clear when using the "Unique visitors" label in the dashboard.

**Comment by ukutaht** [comment]
Your understanding is correct @marcoancona and I would agree that 'unique visitors' can mislead people to think we're tracking uniques across a month, for example.

I've thought about changing a bit how we display the data and call everything 'unique visits' or something like this. But that wouldn't be entirely accurate either because we would still de-duplicate multiple visits within one day.

What would you suggest @marcoancona?

**Reply by marcoancona** [reply]
Technically that counter is the "Sum of daily unique visitors". I understand this is quite verbose. Maybe a tooltip with an explanation placed right next to the "Unique visitors" label would help.

**Comment by toshanmugaraj** [comment]
I know that my organization total no. people using the application is 400, but plausible shows that unique visitors is 585. That is 46.25 % higher. 

Is there a way to feed this deviation to the system so the unique visitors displayed can be approximated.

**Comment by FINDarkside** [comment]
It's pretty inconvenient that right now you have to have a briefing sessions with everyone you give access to Plausible because it's pretty embarrassing if they start publicly stating that we had X unique visitors in the past year while the real number could be only 0.5% of that. And it's really not the fault of the Plausible user either, since Plausible did incorrectly claim that we had X unique visitors in the past year.

The previous argument for not renaming is that people are used to "unique visitors" stat. But I think that's not a great argument since Plausible does not have such stat. Mislabeling another stat to "unique visitors" doesn't really help.

**Comment by millieone** [comment]
I just now was trying to figure out why my unique visitors are so much higher than google.

You definitely shouldn't call them "unique visitors" when the time period is more than a day because it is misleading.

But this seems to be the biggest limitation of plausible. Probably the #1 metric we want is how many unique visitors are visiting the site over a certain time period.

Why not get rid of the daily salt? Can I pay to get rid of it?

Or at the very least have a monthly salt option?  Unique visitors per month is a basic metric. Beyond that timeframe we don't report it and haven't seen it reported by others. But can we get a monthly salt?

**Reply by marcoancona** [reply]
I guess the problem here is, because most consumer IPs are not static, the probability that the same user comes with different IPs on different days is high. If this happens, those will be counted as two users, independent of whether the daily salt is there or not. I don't think there is a bullet-proof way to count unique visitors without storing a cookie.

**Reply by millieone** [reply]
But a cookie isn't perfect either if someone switches browsers. and if I'm on my phone and laptop I count twice with a cookie.  The month long salt would at least give some idea of a monthly user account even if its not as accurate as a cookie count.

Still don't understand the problem with cookies but I'd use GA for 17 years.

**Reply by marcoancona** [reply]
Indeed cookies also don't help cross-device. IP does not either if the phone is not connected via the same WIFI. Additionally, Plausible uses the browser user agent in the signature to identify users, so phone and desktop would always be seen as separate users, even if they have the same IP and they had no salt. I guess the reason is that there are often multiple people connected to the same IP so you need to avoid counting them all as the same visitor cause you'd risk the opposite: to underestimate visitors.

**Comment by anilopez** [comment]
I agree that if there is going to be just one of the three (MAU, WAU, DAU) there should be a way con configure it for salt deletion period that fits your business or legal compliance better. Example: European Digital Services Act (DSA) asks to inform MAU in some types of sites.
